packages/mask/src/plugins/WalletService/services/wallet/index.ts

```diff
@@ -162,14 +163,8 @@ export async function renameWallet(address: string, name: string) {
     })
 }
 
-export async function removeWallet(address: string, unverifiedPassword: string) {
-    await password.verifyPasswordRequired(unverifiedPassword)
-
-    // delete a wallet with derivationPath is not allowed
-    const wallet = await database.getWalletRequired(address)
-    if (wallet.derivationPath) throw new Error('Illegal operation.')
-
-    await database.deleteWallet(wallet.address)
+export async function removeWallet(address: string, unverifiedPassword?: string) {
+    await database.deleteWallet(address)
```

Comment:
For a dangerous action, must verify the password.

Reply:
User has decided to do  this dangerous action.